cmd/root.go

```diff
@@ -107,4 +108,12 @@ func init() {
 		"o",
 		"Output file format. One of: "+strings.Join(output.AllowedValues(), ", "),
 	)
+
+	rootCmd.Flags().BoolVarP(
+		&layers,
+		"layers",
+		"s",
```

Comment:
```suggestion
	rootCmd.Flags().BoolVar(
		&layers,
		"layers",
```

Reply:
I can not remove the "s".
![Screenshot from 2022-05-16 13-18-13](https://user-images.githubusercontent.com/35694200/168657170-0021cea8-714f-4565-96fc-06ff595fea27.png)
It seems the rootCmd needs a letter apart from the --command

Reply:
Please note that line 112 changed as well 🙂

Reply:
Done in #[06fc97c](https://github.com/patrickhoefler/dockerfilegraph/pull/88/commits/06fc97c38c261473ee5bb9273041c36363b215af)